src/game/frontend/manager/Widgets/LoopedCommand.cpp

```diff
@@ -47,7 +47,7 @@ namespace YimMenu
 
 		if (ImGui::IsItemHovered())
 		{
-			ImGui::SetTooltip(std::format("{}\n\nPress shift to assign a hotkey", Command->GetDescription()).data());
+			ImGui::SetTooltip(std::format("{}\n\nPress shift to assign a hotkey", Command->GetDescription()).data()); // TODO: do we need this?
```

Comment:
Which part are you referring to? The actual description or the button suggestion? Or the fact there is a popup at all?